Xamarin.Forms.ControlGallery.Android/CustomRenderers.cs

```diff
@@ -564,6 +566,95 @@ protected override NumberKeyListener GetDigitsKeyListener(InputTypes inputTypes)
 	}
 
 
+
+#pragma warning disable 618
```

Comment:
Instead of disabling the warning, just add the missing constructor for these. That way if people are looking here for example code they don't copy the obsolete renderer format.